ui/lib/kv/addon/templates/list-directory.hbs

```diff
@@ -6,4 +6,6 @@
   @filterValue={{this.model.filterValue}}
   @noMetadataListPermissions={{this.model.noMetadataListPermissions}}
   @breadcrumbs={{this.breadcrumbs}}
+  @routeName={{this.routeName}}
+  @meta={{this.model.secrets.meta}}
```

Comment:
Not sure why we need to pass this separately if it's present on `@secrets` which is already passed in. To me, accessing from `@secrets.meta` makes it clear that the metadata is related directly to that passed model

Reply:
That makes sense to me. It was a left over for me as I was trying to keep track of things when originally playing around with the component.